moped-data-events/activity_log/MopedEvent.py

```diff
@@ -47,7 +47,7 @@ class MopedEvent:
           }) {
             affected_rows
           }
-          update_moped_project(where: {project_id: {_eq: $recordId}}, _set: {updated_at: $timestamp}) {
+          update_moped_project(where: {project_id: {_eq: $recordProjectId}}, _set: {updated_at: $timestamp}) {
```

Comment:
This would have worked when we were looking for updates on the project table when we were making *edits* to the project table, as both parts of this mutation were being fed the right `recordId`, but it would not have been working for other tables that are related to project by a FK relationship.